### PR TITLE
web: display games

### PR DESCRIPTION
**Pull request by Cari1111**
# Issus

Helps to solve #112 

# Changes

Added a page that can display all the relevant information for games from the database under `http://192.168.178.25:3000/game/[game_id]`. 
Added new database functions for this.

# Still TODO

The download button to download actions is still a dummy button that downloads a dummy file.

### CODE REVIEW

File: comprl-web/app/routes/_dashboard.game.$id.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+          <DashboardPaper>
+            <Typography variant="h5" marginBottom={2}> Game Information </Typography>
+            <Typography variant="body1"> Time Stamp: {game.start_time} </Typography>
+            <Typography variant="body1"> Game End: {gameEnd} </Typography>
+            {game.participants.map( (participant, i) => {
+              if (participant.winner) return <Typography variant="body1"> Winner: {participant.name} </Typography>
+              return null
+            })}
+            {game.participants.map( (participant, i) => {
+              if (participant.disconnected) return <Typography variant="body1"> Winner: {participant.name} </Typography>
```
Comment (PacoSchatz) [inline comprl-web/app/routes/_dashboard.game.$id.tsx:86]: `Winner` should be `Disconnected player` or something like that. Seems like you thought of it but forgot to actually change the word ;)

**Review by PacoSchatz — CHANGES_REQUESTED** [review]
Nicely done, thank you. I really like your layout and it works beautifully on different screen sizes. The code looks also good to me. One little typo I discovered was when looking at a game with a disconnect.

And yeah, the download button can be a new issue.

File: comprl-web/app/routes/_dashboard.game.$id.tsx (hunk 1 above)
Comment (Cari1111) [reply comprl-web/app/routes/_dashboard.game.$id.tsx:86]: Oh, I must of missed that, thanks

Comment (PacoSchatz) [reply comprl-web/app/routes/_dashboard.game.$id.tsx]: Happens to the best. Thx for changing it

**Comment by PacoSchatz** [comment]
Neat addition of the icons visualizing the end state. I like it 👍

**Review by PacoSchatz — APPROVED** [review]

**Pull request merged**